README.md

```diff
@@ -48,6 +48,8 @@ ZCBOR_STATE_D(decode_state, n, payload, payload_len, elem_count);
 ZCBOR_STATE_E(encode_state, n, payload, payload_len, 0);
 ```
 
+The CBOR libraries assume little-endianness by default, but you can define ZCBOR_BIG_ENDIAN to change this.
```

Comment:
Is it given _how_ you would define it? Can it be passed to zcbor as an argument or do you just define it yourself? If its the latter then I guess the current description would suffice.

Reply:
We should add more info about configuration. I made a task.